mdssdk/switch.py

```diff
@@ -54,14 +55,17 @@ class Switch(SwitchUtils):
 
     :example:
         >>> switch_obj = Switch(ip_address = switch_ip, username = switch_username, password = switch_password)
+        >>> switch_obj = Switch(ip_address = switch_ip, username = switch_username, use_keys = True, key_file = './ssh/test_rsa')
```

Comment:
Why do we need `use_keys`? if `key_file` is present then we can ignore `password`
Seems redundant to pass both `use_keys`and `key_file`

Reply:
fixed...